model/pokemon.go

```diff
@@ -0,0 +1,16 @@
+/*
+Package model defines the structure of an object.
+*/
+package model // import "go-bootcamp/model"
+
+// Pokemon is the model which represents a Pokemon object
+type Pokemon struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+// PokemonColor is the model which represents a Pokemon object and its color
+type PokemonColor struct {
+	Pokemon Pokemon `json:"data"`
+	Color   string  `json:"color"`
```

Comment:
This does not make sense, Shouldn't color be part of the Pokemon? instead, you are encapsulating a Pokemon by color, but as you know there are a lot of Pokemon with the same color.